content/blog/unlocking-spinnaker-with-pulumi/index.md

```diff
@@ -23,4 +23,8 @@ Here's a brief overview of the plugin.
 
 ![Pulumi logs](./spinnaker-pulumi-preview.png)
 
+Here's a demo of the plugin:
```

Comment:
@isaac-pulumi I went through a few variations of what to say here in my mind and felt that keeping this succinct was the best. Let me know if you would like me to say something else here.